doc/lsp.txt

```diff
@@ -380,6 +380,18 @@ completionMatcher	|String| option.  Enable fuzzy or case insensitive
 			    case  - case sensitive matching (default).
 			    fuzzy - fuzzy match completion items.
 			    icase - ignore case when matching items.
+						*lsp-opt-noAdditionalTextEdits*
+completionTextEdit	|Boolean| option.  If true, apply the LSP server
```

Comment:
Can you change `lsp-opt-noAdditionalTextEdits` to `lsp-opt-completionTextEdit`?

Reply:
done.